rdkit/Dbase/DbModule.py

```diff
@@ -11,17 +11,17 @@
 from rdkit import six
 from rdkit import RDConfig
 
-if hasattr(RDConfig, "usePgSQL") and RDConfig.usePgSQL:
-  from pyPgSQL import PgSQL
+if hasattr(RDConfig, "usePgSQL") and RDConfig.usePgSQL:  # pragma: nocover
+  from pyPgSQL import PgSQL  # @UnresolvedImport
```

Comment:
Jokes aside: I know what they are, and I think we agreed in the email thread that they wouldn't be showing up in code any more.